tests/storage/backends/test_inmem.py

```diff
@@ -28,7 +29,7 @@ def test_can_schedule_single_job(self, defaultbackend, simplejob):
         new_job = defaultbackend.get_job(job_id)
 
         # Does the returned job record the function we set to run?
-        assert str(new_job.func) == str(id)
+        assert str(new_job.func) == stringify_func(id)
```

Comment:
Not sure what the issue here is. In here, I schedule a new job that just runs the  built in`id` function, so I can have a job that's guaranteed to be able to import said function.

If what you're asking is if I shadow the top level `id` function, nope! It's not currently shadowed by any fixture so far.